build.sbt

```diff
@@ -1,77 +1,86 @@
+import sbt.Keys._
 import ReleaseTransformations._
 import Dependencies._
 
-lazy val commonSettings = Seq(
-  organization := "com.gu",
-  scalacOptions ++= Seq("-feature", "-deprecation", "-target:jvm-1.8", "-language:higherKinds"),
-  scalaVersion := "2.12.4",
-  libraryDependencies ++= coreDeps,
-  dependencyOverrides += "org.apache.thrift" % "libthrift" % "0.9.1",
-  unmanagedResourceDirectories in Compile += (baseDirectory in ThisBuild).value / "build",
-  excludeFilter in Compile in unmanagedResources := "*.fjs" || "*.scss"
-)
-
-lazy val coreSettings = Seq(
-  name := "atom-renderer",
-  crossScalaVersions := Seq("2.11.11", "2.12.4"),
-  releaseCrossBuild := true,
-  releasePublishArtifactsAction := PgpKeys.publishSigned.value,
-  publishTo := Some(
-    if (isSnapshot.value)
-      Opts.resolver.sonatypeSnapshots
-    else
-      Opts.resolver.sonatypeStaging
-  ),
-  releaseProcess := Seq[ReleaseStep](
-    checkSnapshotDependencies,
-    inquireVersions,
-    runClean,
-    runTest,
-    setReleaseVersion,
-    commitReleaseVersion,
-    tagRelease,
-    releaseStepCommand("publishSigned"),
-    setNextVersion,
-    commitNextVersion,
-    releaseStepCommand("sonatypeReleaseAll"),
-    pushChanges
-  )
-)
-
-lazy val publishSettings = Seq(
-  sonatypeProfileName := "com.gu",
-  publishMavenStyle := true,
-  licenses := Seq("APL2" -> url("http://www.apache.org/licenses/LICENSE-2.0.txt")),
-  homepage := Some(url("https://(your project url)")),
-  scmInfo := Some(
-    ScmInfo(
-      url("https://github.com/guardian/atom-renderer"),
-      "scm:git@github.com:guardian/atom-renderer.git"
-    )
-  ),
-  developers := List(
-    Developer(id="regiskuckaertz", name="Regis Kuckaertz", email="regis.kuckaertz@theguardian.com", url=url("https://github.com/regiskuckaertz"))
+lazy val root = (project in file("."))
+  .aggregate(core, email, utils)
+  .settings(commonSettings, disablePublishingSettings)
```

Comment:
I didn't know you could pass multiple sequences as parameters to this method. Nice!